### PR TITLE
Modernization

### PR DESCRIPTION
**Pull request by wagenet**
This brings everything up to the latest dependencies and ember-cli blueprint recommendations.

## Open Questions:

* Is it correct to remove bower?
* Should we test against even older Ember versions?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +62,9 @@
   "peerDependencies": {
     "ember-cli-babel": "^6.7.1"
```
Comment (kanongil) [inline package.json]: Shouldn't this be changed to accommodate babel@7?

Comment (wagenet) [reply package.json]: Yes, it should, I had intended to!

**Comment by alexlafroscia** [comment]
> Is it correct to remove bower?

I think this is fine

> Should we test against even older Ember versions?

Going back to `2.12` seems like a reasonable level of coverage to me!

**Comment by btecu** [comment]
This look great!
@stefanpenner can we get a new release with this?

**Comment by wagenet** [comment]
@stefanpenner is there still a desire to support Node 4?

**Comment by stefanpenner** [comment]
we can drop node 4, it will just require a major version bump.

**Pull request closed without merging**